spec/support/playwright_global_setup.js

```
@@ -0,0 +1,34 @@
+// Global setup for Playwright tests in Rails 8 application
+const { execSync } = require('child_process');
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)